src/components/notification/payButton.js

```diff
@@ -26,7 +26,7 @@ export default function PayButton ({finalPrice, tax, tax_base, spaceTitle, tenan
             extra4: spaceId,
             extra5: tenantName,
             extra6: notification._id,
-            response: `http://127.0.0.1:3000/response`,
+            response: `localhost:3000/response`,
```

Comment:
don´t burn the address in the code. Use the environment variables (process.env.REACT_APP_URL)